Documentation/specs/runtime-configuration-file.md

```diff
@@ -2,206 +2,155 @@
-  * **Note:** This section will not be used for standalone applications that do not rely upon a shared framework.
-
-* Others _TBD_
+  * **Note:** This does not apply to `pre-release` versions; it applies only to `production` releases.
+  * **Note:** This section should not exist self-contained applications because they do not rely upon a shared framework.
+* `rollForwardOnNoCandidateFx` - Determines roll-forward behavior of major and minor. Only applies to `production` releases. Values: 0(Off), 1 (roll forward on [minor]), Roll forward on [major] and [minor]
+ See [roll-forward-on-no-candidate documentation](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md) for more information.
 
-These settings are read by `corehost` to determine how to initialize the runtime. All versions of `corehost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
+These settings are read by `apphost` to determine how to initialize the runtime. All versions of `apphost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
 
 ### `compilationOptions` Section (`.deps.json`)
 
-This section is copied by storing the merged `compilationOptions` from the input `project.json`. The `project.json` can define three sets of compilation options: Global, Per-Configuration, and Per-Framework. However, the `[appname].runtimeconfig.json` is specific to a configuration and framework so there is only one merged section here.
-
-The exact settings found here are specific to the compiler that produced the original application binary. Some example settings include: `defines`, `languageVersion` (C#/VB), `allowUnsafe` (C#), etc.
-
-As an example, here is a possible `project.json` file:
-
-```json
-{
-    "compilationOptions": {
-        "allowUnsafe": true
-    },
-    "frameworks": {
-        "net451": {
-            "compilationOptions": {
-                "defines": [ "DESKTOP_CLR" ]
-            }
-        },
-        "dnxcore50": {
-            "compilationOptions": {
-                "defines": [ "CORE_CLR" ]
-            }
-        }
-    },
-    "configurations": {
-        "Debug": {
-            "compilationOptions": {
-                "defines": [ "DEBUG_MODE" ]
-            }
-        }
-    }
-}
-```
-
-When this project is built for `dnxcore50` in the `Debug` configuration, the outputted `MyApp.deps.json` file will have the following `compilationOptions` section:
-
-```json
-{
-    "compilationOptions": {
-        "allowUnsafe": true,
-        "defines": [ "CORE_CLR", "DEBUG_MODE" ]
-    }
-}
-```
+This section is created during build from the project's settings. The exact settings found here are specific to the compiler that produced the original application binary. Some example settings include: `defines`, `languageVersion` (C#/VB), `allowUnsafe` (C#), etc.
```

Comment:
"`languageVersion` (C#/VB)" makes it sound like C# or VB is the possible values for these setting, when in reality it is a number like `7.3`

Reply:
Changing to "(e.g. the version of C# or VB)"